src/commands/Minion/create.ts

```diff
@@ -136,6 +136,9 @@ export default class extends BotCommand {
 		);
 
 		if (!createableItem.noCl) msg.author.addItemsToCollectionLog(outItems);
+		if (createableItem.xpGain && createableItem.xpSkill) {
+			await msg.author.addXP(createableItem.xpSkill, createableItem.xpGain);
```

Comment:
```suggestion
			await msg.author.addXP(createableItem.xpSkill, createableItem.xpGain * quantity);
```
multiple items can be created at the same time